workflow/executor/resource.go

```diff
@@ -265,6 +274,12 @@ func (we *WorkflowExecutor) SaveResourceParameters(resourceNamespace string, res
 		if param.ValueFrom == nil {
 			continue
 		}
+		if ignoreNotFound && resourceName == "" {
```

Comment:
Also, do you mean
```suggestion
		if we.Template.Resource.IgnoreNotFound && resourceName == "" {
```
here?

Reply:
Yes, I did